skyportal/handlers/api/gcn_tach.py

```diff
@@ -0,0 +1,335 @@
+                    gcn_aliases.append(new_gcn_alias)
+            gcn_event.aliases = gcn_aliases
+
+        if gcn_event.circulars is None:
+            gcn_event.circulars = new_gcn_circulars
+        else:
+            gcn_event.circulars = {**gcn_event.circulars, **new_gcn_circulars}
+
+        session.commit()
+
+    flow.push(
+        user_id='*',
+        action_type='skyportal/REFRESH_GCNEVENT',
+        payload={'gcnEvent_dateobs': dateobs},
+    )
+
+
+class GcnTachHandler(BaseHandler):
+    @permissions(["Manage GCNs"])
+    def post(self, dateobs):
+        """
+        ---
+        description: Scrape aliases of a GCN Event from GCNs notice/circulars
+        tags:
+          - gcn_event
+        parameters:
+          - in: path
+            name: dateobs
+            required: true
+            schema:
+              type: string
+            description: The dateobs of the event, as an arrow parseable string
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Success'
+                    - type: object
+                      properties:
+                        data:
+                          type: object
+                          properties:
+                            id:
+                              type: int
+                              description: The id of the GcnEvent
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+        try:
+            arrow.get(dateobs).datetime
+        except Exception:
+            log(f'Invalid dateobs: {dateobs}')
+            return self.error(f'Invalid dateobs: {dateobs}')
+        try:
+            with self.Session() as session:
+                stmt = GcnEvent.select(session.user_or_token).where(
+                    GcnEvent.dateobs == dateobs
+                )
+                gcn_event = session.scalars(stmt).first()
+                if gcn_event is None:
+                    return self.error(f'No GCN event found for {dateobs}')
+
+                tags = gcn_event.tags
+                tach_id = (
+                    gcn_event.tach_id
+                    if gcn_event.tach_id is not None
+                    else get_tach_event_id(dateobs, tags)
+                )
+                if tach_id is None:
+                    return self.error(
+                        f'Event {dateobs} not found on TACH, cannot retrieve aliases'
+                    )
+                gcn_event_id = gcn_event.id
+
+                IOLoop.current().run_in_executor(
+                    None,
+                    lambda: post_aliases(dateobs, tach_id, session.user_or_token),
+                )
+
+                return self.success(data={'id': gcn_event_id})
+
+        except Exception as e:
+            log(f'Error scraping GCN aliases: {e}')
```

Comment:
This one either. 
BTW if these logging commands were added just for debugging, I suggest using a simple `print` which is easier to catch and get rid of before moving to production. 
In general we want to reduce the amount of logging we are doing to only what is really necessary.